mle/agents/advisor.py

```diff
@@ -102,11 +102,12 @@ def __init__(self, model, console=None):
         self.sys_prompt += self.json_mode_prompt
         self.chat_history.append({"role": 'system', "content": self.sys_prompt})
 
-    def suggest(self, requirement):
+    def suggest(self, requirement, process=True):
         """
         Handle the query from the model query response.
         Args:
             requirement: the user requirement.
+            process: whether to process the report.
```

Comment:
sry. What does this process mean?
coud you give an e.g. in this function doc?
as `process the report` is too general for me